src/psygnal/_group_descriptor.py

```diff
@@ -162,7 +176,9 @@ def _build_dataclass_signal_group(
         # patch in our custom SignalInstance class with maxargs=1 on connect_setattr
         sig._signal_instance_class = _DataclassFieldSignalInstance
 
-    return type(f"{cls.__name__}SignalGroup", (SignalGroup,), signals)
+    # Create `signal_group_class` subclass with the attached signals
+    group_name = f"{cls.__name__}_{signal_group_class.__name__}"
```

Comment:
```suggestion
    group_name = f"{cls.__name__}{signal_group_class.__name__}"
```

should be camel case right?

Reply:
Yes!